src/pages/react/training/workshops/design-system-styling-in-react/index.js

```diff
@@ -33,7 +33,7 @@ import { LIST_TWO_COL } from 'src/components/curriculum/selectCurriculumLayout'
 
 const StylingDesignSystemWorkshop = () => (
```

Comment:
@ric9176 StylingDesignSystemWorkshop?

Reply:
I guess bc it was initially copied, I'll go through and update the naming ;-)

Reply:
Yeah, that's why